apps/src/applab/designMode.js

```diff
@@ -180,7 +192,9 @@ designMode.updateProperty = function(element, name, value) {
     case 'style-width':
       var newWidth = appendPx(value);
       element.style.width = newWidth;
-      element.parentNode.style.width = newWidth;
+      if (isDraggableContainer(element.parentNode)) {
+        element.parentNode.style.width = newWidth;
+      }
```

Comment:
OK, I see you've wrapped all the places we would modify the parent's style, so your change looks good. I was also confused about why it's not necessary to set parent width/height, but apparently the DOM takes care of us for that, even when parent and child are both `position: absolute`.